### ISSUE TITLE
replica-disconnected css class

### ISSUE THREAD

**Issue by tkachuk-labs** [pkamenarsky/replica#21]
Maybe it make sense to apply `replica-disconnected` class to root component in case of server disconnect? Similar to this
https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#module-loading-state-and-errors
Then at least client will be notified about disconnect (we can have some fancy semi-transparent blurry overlay with spinner or something like this). At the moment page just freezes and nothing happens and there is no any user-friendly indication what something is going wrong (at least in concur-replica examples).

Probably another issue, but it's nice to try automatically reconnect as well (without page reload).

**Comment by pkamenarsky** [comment]
Yep, there's a [fork](https://github.com/kamoii/replica) of Replica which implements client reconnection, iirc. The general idea is to hold a client's state for some time and resume the session if a client reconnects during that interval.

I haven't gotten around to merging the fork yet, but maybe you'll find it helpful.
